### ISSUE TITLE
Charters Towers Regional Council

### ISSUE THREAD

**Issue by mlandauer** [planningalerts-scrapers/issues#79]
This issue has been **automatically** created by PlanningAlerts for [Charters Towers Regional Council](http://www.planningalerts.org.au/authorities/charters_towers)

It uses the scraper [planningalerts-scrapers/charters_towers](https://morph.io/planningalerts-scrapers/charters_towers)

Only close this issue once the authority is working again on PlanningAlerts. Otherwise a new issue will just automatically created. Also, if there is a duplicate issue close the other one in favour of this one.

**Comment by mlandauer** [comment]
It looks like the page where development applications are shown has moved to https://www.charterstowers.qld.gov.au/services/planning-and-development/decision-notices. No idea if it's still shown in a similar format

**Comment by mlandauer** [comment]
On the page https://www.charterstowers.qld.gov.au/services/planning-and-development/decision-notices there are no applications on exhibition right now so it's hard to know if this is useful. So, labeling as "blocked by authority".

**Comment by mlandauer** [comment]
Still nothing being shown on the page

**Comment by kat** [comment]
No change at https://www.charterstowers.qld.gov.au/services/planning-and-development "Public Notification & Decision Notices" page


I'm wondering if the DAs on notice sit under Decisions because there's nothing under the first heading "Current Development Applications under Public Notification"


hmm 

—
Katherine Szuminska

--- original message ---
On May 30, 2021, 7:40 AM GMT+10 ***@***.*** wrote:



Still nothing being shown on the page

—

You are receiving this because you are subscribed to this thread.

Reply to this email directly, view it on GitHub, or unsubscribe.
--- end of original message ---

**Comment by jamezpolley** [comment]
https://www.charterstowers.qld.gov.au/development-applications-1 has details of applications currently being considered.

Not all of these are open for advertisement though.

https://www.charterstowers.qld.gov.au/public-notification - has address + dates of public advertisement. Application ID is in the URL: `https://www.charterstowers.qld.gov.au/downloads/file/2623/notice-of-intention-to-commence-public-notification-9-october-2023-to-30-october-2023-ral2023-0008-subdivision-1-into-2-black-jack-road-black-jack-qld-4820` -> `ral2023-0008`

**Comment by jamezpolley** [comment]
I don't think this is blocked any more - just needs some work to pull in data. I'm not sure if that should be just the advertised ones or if we want to pull in all DAs and update advertisement dates if/when they appear.

**Comment by katska** [comment]
Development applications  are at https://www.charterstowers.qld.gov.au/development-applications-4
There are separate pages for 2024 and 2025 applications listed.

**Comment by ianheggie-oaf** [comment]
Marked "new scraper required" as its basically a rewrite since its new software.

**Comment by benrfairless** [comment]
Rewrote the scraper for the council's OpenCities site: planningalerts-scrapers/charters_towers#5 (draft). It walks the [All Development Applications](https://www.charterstowers.qld.gov.au/Services/Planning-and-development/Planning-services/All-development-applications) year pages (2017-2026) and captures 218 applications spanning 2020-2026. Pages from 2022 onwards are fully structured (one accordion per application with reference, description and address); 2020-2021 are decision notice PDF lists with addresses in the link text, so those are captured with the application type as a fallback description. Most 2017-2019 entries publish only a reference with no address, so they can't be turned into usable records without parsing the PDFs. The received date isn't published anywhere, so date_received is not set. The site is behind Akamai and needs browser-like headers including the Sec-Fetch-* set, which the scraper sends. Still to do after merge: confirm the morph.io run succeeds on heroku-18, and check the PlanningAlerts authority record still points at the charters_towers morph scraper with data flowing again before closing this issue.
